include/revng/Pypeline/MapContainer.h

```diff
@@ -0,0 +1,72 @@
+#pragma once
+
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+#include "revng/Pypeline/Common.h"
+#include "revng/Pypeline/ObjectID.h"
+
+namespace revng::pypeline {
+
+namespace detail {
+
+template<ObjectID::Kind KindParam, ConstexprString NameParam>
```

Comment:
Remove `Param`. Instead, use `TheKind` here or below.

Reply:
Done for both `KindParam` and `NameParam`